### PR TITLE
Hello Microverse Project PR

### PR DESCRIPTION
**Pull request by tarek102**
Added index.html
Added styles.css
Installed linters
Checked for linter errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
+# hello-world
```
Comment (Kossi-stack) [inline README.md]: - [x] Kindly remove the hello-word to Hello Microverse.

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
 
+> Setting up linters.
+
```
Comment (Kossi-stack) [inline README.md:4]: - [x] Kindly remove all the empty lines in the readme file for a clean code.

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+- Go to the [repository page](https://github.com/tarek102/hello-world)
+- Click on code and copy the link
+- Go to command line and type <git clone https://github.com/tarek102/hello-world.git>
+
+
+
+
+## Authors
+
+👤 **Author1**
```
Comment (Kossi-stack) [inline README.md]: - [ ] Kindly remove Author1 by your name

File: README.md [hunk 4]
```diff
@@ -1 +1,50 @@
...
+
+- GitHub: [@tarek102]((https://github.com/tarek102))
```
Comment (Kossi-stack) [inline README.md]: - [ ] Kindly fix the link by removing one parenthesis (), once you remove it work.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @tarek102,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [x] Kindly edit the PR's title to something more descriptive. it can be "Hello Microverse Project PR or Hello Microverse Project".

 _Check the comments under the review for more changes_.

## Optional suggestions
 - [ ] Kindly add descriptive pull request message, This helps other programmers who want to take a look at your code understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zhadier — APPROVED** [review]
## Hi @tarek102  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**